docs/get-must-not-be-lro.md

```diff
@@ -0,0 +1,70 @@
+# GetMustNotBeLRO
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-14
+
+## Output Message
+
+The Get call cannot be Long Running Operation and it must not have `x-ms-long-running-operation-options` property block.
+
+## Description
+
+Only asynchronous(i.e. Long Running Operation) can have `x-ms-long-running-operation-options` property.
+The Get calls are synchronous and it MUST NOT have
+    - `x-ms-long-running-operation-options` property block
```

Comment:
>block [](http://example.com/codeflow?start=53&length=5)

I dont think its a block, just a property?

Reply:
x-ms-long-running-operation is property, but x-ms-long-running-operation-options is block